README.md

```diff
@@ -2,7 +2,7 @@
 
 React is a JavaScript library for building user interfaces.
 
-* **Declarative:** React makes it painless to create interactive UIs. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes. Declarative views make your code more predictable, simpler to understand, and easier to debug.
```

Comment:
This is targetted at the reader. The change alters the meaning and doesn't match the ", and React will" part of the sentence.